interfaces/seccomp/backend.go

```diff
@@ -81,6 +83,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err != nil {
 		return fmt.Errorf("cannot synchronize security files for snap %q: %s", snapName, err)
 	}
+
+	for baseName := range content {
+		in := filepath.Join(dirs.SnapSeccompDir, baseName)
+		out := filepath.Join(dirs.SnapSeccompDir, baseName+".bpf")
```

Comment:
Thanks for the explanation. I forgot we'll always rebuild the profile on startup due to the binary change. It doesn't seem worth adding extra complexity to preserve the directory when something simpler along the lines of what you've done will sort it out just fine.

Here is an alternative suggestion: _seccomp/bpf/*.src_ for the source files and _.../*.bin_ for the binary ones. The _.src_ is just a trivial hint to suggest that this file won't really be used by the runtime until it is compiled. The _.bin_ is a way to avoid the _.../bpf/*.bpf_ duplicity, and only really recommended if there's no standard suffix for the BPF bytecode (I couldn't find .bpf as a traditional suffix.. is it?).